### DISCUSSION TITLE
Please add snap turn

### DISCUSSION THREAD

**Discussion by ghost** [praydog/REFramework#958] [General Questions]
I can't play any game with smooth turn. Can you add snap turn please? Talking mainly about RE2

**Comment by maxsteep** [comment]
yes please

**Comment by MonoksideC** [comment]
That would be a dream come true.  Smooth turning in VR makes me feel like I'm on a merry go round.

**Reply by GennadijK1987** [reply]
Want it too!

**Comment by Diceman0925** [comment]
If you're using a keyboard it is X.

**Comment by llmgptuser** [comment]
Use https://github.com/llmgptuser/RE2-3-Snap-Turn for RE2/3

**Reply by saulojoab** [reply]
trying to use this, I see the in-game menu and have the snap turning checkbox enabled but it seems to have no effect. 
the game still uses smooth turning. 

do you know what that could be? tried with VD + steamVR and also directly with steamVR.
